README.md

```diff
@@ -199,6 +200,80 @@ Although these arguments are optional, a (\*) indicates a recommended parameter
 | `hunkUrlTemplate`          | If provided, LaunchDarkly will attempt to generate links to your Git service provider per code reference. Example: `https://github.com/launchdarkly/ld-find-code-refs/blob/${sha}/${filePath}#L${lineNumber}`. Allowed template variables: `sha`, `filePath`, `lineNumber`. If `hunkUrlTemplate` is not provided, but `repoUrl` is provided and `repoType` is not custom, LaunchDarkly will automatically generate links to the repository for each code reference.      |                                |
 | `version`                  | If provided, the current `ld-find-code-refs` version number will be logged, and the scanner will exit with a return code of 0.                                                                                                                                                                                                                                                                                                                                           | `false`                        |
 
+### Configuring aliases
+
+Flag key aliases may be defined using a YAML file stored in your repository at `.launchdarkly/config.yaml`. Configuration types may be used in conjunction and defined more than once for comprehensive alias coverage.
+
+
+#### Hardcoded map of flag keys to aliases
+
+Example hardcoding aliases for a couple flags:
+
+```yaml
+aliases:
+  - type: literal
+    flags:
+      my-flag:
+        - myFlag
+        - isMyFlagOn
+      my-other-flag:
+        - other.flag.alias
+```
+
+#### Flag keys transposed to common casing conventions
+
+Example string: `AnyKind.of_key`
+
+| Type             | After      |
+|------------------|------------|
+| `camelCase`      | `anyKind.ofKey`  |
+| `pascalCase`     | `AnyKind.OfKey`  |
+| `snakeCase`      | `any_kind.of_key` |
+| `upperSnakeCase` | `ANY_KIND.OF_KEY` |
+| `kebabCase`      | `any-kind.of-key` |
+| `dotCase`        | `any.kind.of.key` |
+
+Example generating aliases in camelCase and PascalCase:
+
+```yaml
+aliases:
+  - type: camelCase
+  - type: pascalCase
+```
+
+#### Search a file for a specific pattern
+
+Specify a file to search, with a regular expression containing a capture group to match aliases. The pattern must contain the templated literal `FLAG_KEY` to be replaced with flag keys.
```

Comment:
what does "templated literal `FLAG_KEY`" mean? does that mean the regex has to have the word `FLAG_KEY` in it and ld-find-code-refs will sub that out for each key?

Reply:
That's right - should do a pass on copy (probably in a separate PR)